internal/runners/initialize/init.go

```diff
@@ -55,6 +84,15 @@ func sanitize(params *RunParams) error {
 		params.version = langParts[1]
 	}
 
+	// Disambiguate "python", defaulting to "python3" if no version was given.
+	if langParts[0] == "python" {
+		if params.version == "" || strings.HasPrefix(params.version, "3") {
+			langParts[0] = "python3"
+		} else if strings.HasPrefix(params.version, "2") {
+			langParts[0] = "python2"
+		}
+	}
```

Comment:
We should use language constants here, eg. `language.Python3.String()`.

Also, since we are already interpreting the input here I think we should map this to a language constant, rather than to a string that we then redundantly infer with `language.MakeByName()`.

Reply:
I made some changes to use constants, but I'm not sure I accurately reflected your request. Please let me know if I missed anything.